packages/docs/components/number-input.md

```diff
@@ -1,69 +0,0 @@
-::: slot nimatron-all
```

Comment:
Confirmation note for other reviewers that this removal is intentional and non-breaking.

Reply:
https://github.com/okta/odyssey/pull/714#discussion_r509602364